_packaging/hook-pydot.py

```diff
@@ -0,0 +1,96 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
```

Comment:
That's a tough one. Normally i would say yes, but in this case I stripped pretty much everything from the (windows) installation. Graphviz has dependencies that require pango and cairo, which conflict with our builds. Also we do not need them, since we're only interested in `dot`-file output (no rendered images).